src/sentry/snuba/discover.py

```diff
@@ -1640,6 +1639,79 @@ def find_histogram_params(num_buckets, min_value, max_value, multiplier):
     return HistogramParams(num_buckets, bucket_size, start_offset, multiplier)
 
 
+def find_span_histogram_min_max(span, min_value, max_value, user_query, params, data_filter=None):
+    """
+    Find the min/max value of the specified span. If either min/max is already
+    specified, it will be used and not queried for.
+
+    :param [Span] span: A span for which you want to generate the histograms for.
+    :param float min_value: The minimum value allowed to be in the histogram.
+        If left unspecified, it is queried using `user_query` and `params`.
+    :param float max_value: The maximum value allowed to be in the histogram.
+        If left unspecified, it is queried using `user_query` and `params`.
+    :param str user_query: Filter query string to create conditions from.
+    :param {str: str} params: Filtering parameters with start, end, project_id, environment
+    :param str data_filter: Indicate the filter strategy to be applied to the data.
```

Comment:
You aren't using `data_filter`.. yet?

Reply:
not yet, I was gonna get more context on it before implementing it. I wonder under what circumstances do we want to exclude outliers for spans histogram. Doesn't it make sense to just show all data?